kube/services/revproxy/gen3.nginx.conf/arborist-service.conf

```diff
@@ -51,6 +51,16 @@ location = /authz/resources {
     proxy_pass $upstream/auth/resources;
 }
 
+location = /authz/mapping {
+    if ($csrf_check !~ ^ok-\S.+$) {
+        return 403 "failed csrf check";
+    }
+
+    set $upstream http://${arborist_release_name}-service.$namespace.svc.cluster.local;
+
+    proxy_pass $upstream/auth/mapping?username=$arg_username;
```

Comment:
what is $arg_username?

Reply:
the `username` query string for the endpoint--if I leave it out then nginx seems to not pass the query args to arborist